units/task_stranded_cleanup.go

```diff
@@ -69,39 +74,54 @@ func (j *taskStrandedCleanupJob) Run(ctx context.Context) {
 		taskIDs = append(taskIDs, h.RunningTask)
 		hostIDs = append(hostIDs, h.Id)
 
-		j.AddError(model.ClearAndResetStrandedHostTask(&h))
+		catcher.Wrapf(model.ClearAndResetStrandedHostTask(&h), "fixing stranded host task '%s' on host '%s'", h.RunningTask, h.Id)
 	}
 
+	grip.InfoWhen(len(taskIDs) > 0 || len(hostIDs) > 0, message.Fields{
```

Comment:
Wouldn't we want `&&` here? Seems like we would only want to log out when we have found tasks to fix.

Reply:
Hmm, I re-read this and checking either `taskIDs` or `hostIDs` being non-empty is redundant. It seems like it's easier to just return early if the query returns no hosts, so we don't need to conditionally log this.